flagsmith/streaming_manager.py

```diff
@@ -48,9 +43,6 @@ def run(self) -> None:
             except requests.exceptions.ReadTimeout:
                 pass
 
-            except (FlagsmithAPIError, requests.RequestException):
```

Comment:
I'm not sure why there are changes to the streaming manager here?

Reply:
It's apparent if you read the PR description?

Reply:
Not to me it's not - this isn't using `update_environment()` ?

Reply:
Ah, or maybe it is - `self.on_event` is set to `update_environment` I guess?

Reply:
Went ahead and opened https://github.com/Flagsmith/flagsmith-python-client/pull/101 with this branch as the target branch so you can see the diff and decide.